generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -199,7 +190,8 @@ Scenario: Running a 'shorterThan' request alongside a non-contradicting longerTh
        | null |
        | "♀♀" |
 
-Scenario: Running a 'shorterThan' request alongside a contradicting longerThan (too long) constraint should fail with an error message
+@ignore
```

Comment:
Can we raise a bug on this or link to an existing?

Reply:
This is being addressed in ticket #501